cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -918,20 +918,20 @@ Usage: cardano-cli conway governance drep delegation-certificate
                                                                    | --stake-script-file FILE
                                                                    | --stake-address ADDRESS
                                                                    )
-                                                                   ( --stake-pool-verification-key STRING
-                                                                   | --cold-verification-key-file FILE
-                                                                   | --stake-pool-id STAKE_POOL_ID
-                                                                   | --drep-verification-key STRING
-                                                                   | --drep-verification-key-file FILE
-                                                                   | --drep-key-hash HASH
-                                                                   | ( --drep-verification-key STRING
+                                                                   ( ( --drep-verification-key STRING
                                                                      | --drep-verification-key-file FILE
                                                                      | --drep-key-hash HASH
                                                                      )
                                                                      ( --stake-pool-verification-key STRING
                                                                      | --cold-verification-key-file FILE
                                                                      | --stake-pool-id STAKE_POOL_ID
                                                                      )
+                                                                   | --stake-pool-verification-key STRING
```

Comment:
We need to rename these parser to indicate what they are doing. It's not obvious.